imagery-ol/plugins/annotations/annotations-context-menu/components/annotation-context-menu/annotation-context-menu.component.less

```diff
@@ -1,6 +1,6 @@
-@import 'less/common';
-@import 'less/colors';
-@import 'less/z-index-globals';
+@import '../../../../../../style/less/common.less';
```

Comment:
better solution?

Reply:
the other solution we talked about (adding path to ansyn) didn't work! So it's ok for now